client/client_test.go

```diff
@@ -41,12 +40,35 @@ func TestNew(t *testing.T) {
 	chainservB := chainservice.NewSimpleChainService(chain, b)
 	messageserviceB := messageservice.NewTestMessageService(b)
 	storeB := store.NewMockStore(bKey)
-	New(messageserviceB, chainservB, storeB, logDestination)
+	clientB := New(messageserviceB, chainservB, storeB, logDestination)
 
 	messageserviceA.Connect(messageserviceB)
 	messageserviceB.Connect(messageserviceA)
-	clientA.CreateDirectChannel(b, types.Address{}, types.Bytes{}, outcome.Exit{}, big.NewInt(0))
+	// Set up an outcome that requires both participants to deposit
+	outcome := outcome.Exit{outcome.SingleAssetExit{
+		Allocations: outcome.Allocations{
+			outcome.Allocation{
+				Destination: types.AddressToDestination(a),
+				Amount:      big.NewInt(5),
+			},
+			outcome.Allocation{
+				Destination: types.AddressToDestination(b),
+				Amount:      big.NewInt(5),
+			},
+		},
+	}}
 
-	<-time.After(time.Second)
+	id, res := clientA.CreateDirectChannel(b, types.Address{}, types.Bytes{}, outcome, big.NewInt(0))
```

Comment:
Here the application is getting back an objective id, and a channel funded event. So we are mixing objective-based and channel-based signals. I'm not sure what we want long term. 

Short term, we only care about the objective completing in this test. So it might be best to fallback to just that for now. We can extend that in future when we know what we want.

Reply:
[56cf7e8](https://github.com/statechannels/go-nitro/pull/208/commits/56cf7e8db6f7b278a425f0579975f938479fac89)